### PR TITLE
feat: adds smart account template

### PR DESCRIPTION
**Pull request by malcodeman**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+package-lock.json
```
Comment (malcodeman) [inline .gitignore:38]: Don't think we need these in gitignore, we use yarn.lock here, also we already have this `.env*.local`

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,3 @@
+NEYNAR_API_KEY=
+PIMLICO_API_KEY=
+PRIVATE_KEY=
```
Comment (malcodeman) [inline .env.example:3]: Perhaps better to add this to README, wdyt?

File: README.md [hunk 3]
```diff
@@ -1 +1,6 @@
 # frames
+
+/simple
```
Comment (malcodeman) [inline README.md:3]: Maybe we should provide more detailed readme file?

File: src/app/api/og/route.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+          </div>
+          
+        </div>
+      ),
+      {
+        width: 1200,
+        height: 630,
+      },
+    );
+  } catch (e: any) {
```
Comment (malcodeman) [inline src/app/api/og/route.tsx:63]: Can we remove any type?

File: yarn.lock
Comment (malcodeman) [inline yarn.lock:1]: Looks like we have too many changes in yarn.lock, can you please delete and re-generate it with `yarn install`

File: README.md (hunk 3 above)
Comment (himanshugarg06) [reply README.md:3]: yes readme I will add after this.

**Pull request merged**